views/js/qtiCreator/widgets/component/minMax/minMax.js

```diff
@@ -360,6 +360,13 @@ define([
 
                     fromField = fromField || fields.min;
 
+                    if (isNaN(this.getMinValue())) {
+                        this.setMinValue(config.lowerThreshold);
+                    }
+                    if (isNaN(this.getMaxValue())) {
+                        this.setMaxValue(config.lowerThreshold);
```

Comment:
**question:** Why is the fallback set with the lower threshold? Shouldn't it be the `upperThreshold` instead?

**suggestion:**
```suggestion
                        this.setMaxValue(config.upperThreshold);
```

Reply:
I was thinking about it but mostly people put there low numbers and after this operation (when user deletes value from input) input will be filed wit max int value. I don't think its super user friendly and intuitive. Sure but still we can change it but I don't like it.